src/Statement/Filter/PropertySetStatementFilter.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+namespace Wikibase\DataModel\Services\Statement\Filter;
+
+use Wikibase\DataModel\Statement\Statement;
+use Wikibase\DataModel\Statement\StatementFilter;
+
+/**
+ * @since 3.2
+ *
+ * @licence GNU GPL v2+
+ * @author Thiemo Mättig
+ */
+class PropertySetStatementFilter implements StatementFilter {
+
+	/**
+	 * @var string[]
+	 */
+	private $propertyIds;
+
+	/**
+	 * @param string[]|string $propertyIds One or more property id serializations.
+	 */
+	public function __construct( $propertyIds ) {
```

Comment:
Whut? Why? How would this look like in production?
```php
$setting = array( 'P1', 'P2' );
$objects = $this->pointlessConversionToObects( $setting );
// Pass to the filter
$strings = $this->pointlessConversionBackToStrings( $objects );
// Use for filtering
```
Can we please **not** do this?